index.js

```diff
@@ -1 +1,2 @@
+'use strict';
```

Comment:
Wouldn't this be removed when the linter runs because it assumes that code files are ran in ES6 environment where strict mode is already applied?

Reply:
these files are CJS and use `require`, they not ESM.

running `npm run eslint --fix` does not remove the `'use strict'`